src/scss/includes/components/shareMenu.scss

```diff
@@ -1,37 +1,33 @@
-.contextMenu {
+.shareMenu {
   position: relative;
-  z-index: 3;
+  overflow: visible;
+  text-align: left;
 
   &-button {
     height: 30px;
     width: 30px;
     display: block;
-    border-radius: 50%;
     text-align: center;
     line-height: 30px;
-    color: $primary_clear;
-
-    &:hover {
-      background: $separator_color;
-      color: $primary_alternative_text;
-      cursor: pointer;
-    }
+    cursor: pointer;
   }
 
   &-menu {
-    position: absolute;
-    right: 36px;
-    top: -15px;
-    padding: 6px 9px;
+    position: fixed;
+    z-index: 4;
+    margin-left: -120px;
+    margin-top: 10px;
     background: $background;
     color: $primary_text;
     font-size: 13px;
     border-radius: 3px;
     box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.2);
+    width: 150px;
   }
 
   &-menuItem {
-    padding: 4px 6px;
+    padding: 7px 12px;
+    height: 32px;
     cursor: pointer;
     
     &:hover {
```

Comment:
A new color `#e5eeff` for `hover` state should be introduced according to the mockups

Reply:
Also, text should adopt `grey-black` color on hover.
I think we should define color variables with names taken from mockups, like `grey-black`.